### PR TITLE
Feature/fixes added pointset

### PR DESCRIPTION
**Pull request by flibbertigibbet**
- Reset some state when toggling between point sets
- Show only the currently selected indicator in the hover text, in case of many

### CODE REVIEW

File: js/OTPALayer.js [hunk 1]
```diff
@@ -265,6 +267,8 @@ L.OTPALayer = L.FeatureGroup.extend({
       // TODO: have total count here as "n"; use for graph totals, if summary
       // (have modified backend to always return all as geojson, instead of summary if > 200)
       self._pointsetData = pointset;
+      self._pointsetLayer.clearLayers();
+      self._filteredPointsetLayer.clearLayers();
```
Comment (maurizi) [inline js/OTPALayer.js:271]: Should probably call `self._highlightedLayer.clearLayers()` here as well.

**Comment by maurizi** [comment]
When I added the 2nd data set, I was able to switch back and forth and everything seems to be working swimmingly

... but, on page load, the dropdown showed `blockgroups.geo`, but the cornerstores dataset was displayed.

**Comment by flibbertigibbet** [comment]
Changed the default dataset, and moved the graph a bit to accommodate some large values.

File: js/OTPALayer.js (hunk 1 above)
Comment (maurizi) [inline js/OTPALayer.js:271]: And should probably clear the info window as well.

**Pull request merged**